src/deviation.jl

```diff
@@ -9,13 +9,10 @@ Count the number of indices at which the elements of the arrays
 `a` and `b` are equal.
 """
 function counteq(a::AbstractArray, b::AbstractArray)
-    n = length(a)
-    length(b) == n || throw(DimensionMismatch("Inconsistent lengths."))
+    length(a) == length(b) || throw(DimensionMismatch("Inconsistent lengths."))
     c = 0
-    for i = 1:n
-        @inbounds if a[i] == b[i]
-            c += 1
-        end
+    for i = eachindex(a, b)
+        c += (a[i] == b[i])
```

Comment:
We could just use
```suggestion
    for (ai, bi) in zip(a, b)
        c += (ai == bi)
```
?